tests/unit/test_contrib/test_sqlalchemy/test_dto_integration.py

```diff
@@ -328,6 +329,121 @@ def get_handler(data: Circle) -> Circle:
         assert module.DIAMETER == 10
 
 
+async def test_dto_with_composite_map() -> None:
+    @dataclasses.dataclass
```

Comment:
minor nitpick why `dataclasses.dataclass` when `dataclass` is already imported?

Reply:
Because I copy pasted the code from SQLAlchemy docs :D, I'll fix it, thanks.